packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3214,7 +3234,7 @@ export default class Meeting extends StatelessWebexPlugin {
       | any,
     destination: object | string | null = null
   ) {
-    const webexMeetingInfo = meetingInfo?.body;
+    const webexMeetingInfo = meetingInfo?.body || meetingInfo;
```

Comment:
We needed the `meetingInfo.body` format because later it uses `meetingInfo.error`

I added an error parameter and moved `parseMeetingInfo` out from `setMeetingInfo` to make it more clear